code/game/objects/items/cardboard_cutouts.dm

```diff
@@ -0,0 +1,175 @@
+		"Ash Walker", "Deathsquad Officer", "Ian", "Slaughter Demon",
+		"Laughter Demon", "Xenomorph Maid", "Private Security Officer")
+	var/pushed_over = FALSE //If the cutout is pushed over and has to be righted
+	var/deceptive = FALSE //If the cutout actually appears as what it portray and not a discolored version
+	var/lastattacker = null
+
+/obj/item/cardboard_cutout/attack_hand(mob/living/user)
+	if(user.a_intent == "help" || pushed_over)
+		return ..()
+	user.visible_message("<span class='warning'>[user] pushes over [src]!</span>", "<span class='danger'>You push over [src]!</span>")
+	playsound(src, 'sound/weapons/Genhit.ogg', 50, 1)
+	push_over()
+
+/obj/item/cardboard_cutout/proc/push_over()
+	name = initial(name)
+	desc = "[initial(desc)] It's been pushed over."
+	icon = initial(icon)
+	icon_state = "cutout_pushed_over"
+	color = initial(color)
+	alpha = initial(alpha)
+	pushed_over = TRUE
+
+/obj/item/cardboard_cutout/attack_self(mob/living/user)
+	if(!pushed_over)
+		return
+	to_chat(user, "<span class='notice'>You right [src].</span>")
+	desc = initial(desc)
+	icon = initial(icon)
+	icon_state = initial(icon_state) //This resets a cutout to its blank state - this is intentional to allow for resetting
+	pushed_over = FALSE
+
+/obj/item/cardboard_cutout/attackby(obj/item/I, mob/living/user, params)
+	if(istype(I, /obj/item/toy/crayon))
+		change_appearance(I, user)
+		return
+	// Why yes, this does closely resemble mob and object attack code.
+	if(I.flags & NOBLUDGEON)
+		return
+	if(!I.force)
+		playsound(loc, 'sound/weapons/tap.ogg', 20, 1, -1)
+	else if(I.hitsound)
+		playsound(loc, I.hitsound, 20, 1, -1)
+
+	user.changeNext_move(CLICK_CD_MELEE)
+	user.do_attack_animation(src)
+
+	if(I.force)
+		user.visible_message("<span class='danger'>[user] has hit \
+			[src] with [I]!</span>", "<span class='danger'>You hit [src] \
+			with [I]!</span>")
+
+		if(prob(I.force))
+			push_over()
+
+/obj/item/cardboard_cutout/bullet_act(obj/item/projectile/P)
+	visible_message("<span class='danger'>[src] has been hit by [P]!</span>")
+	playsound(src, 'sound/weapons/slice.ogg', 50, 1)
+	if(prob(P.damage))
+		push_over()
+
+/obj/item/cardboard_cutout/proc/change_appearance(obj/item/toy/crayon/crayon, mob/living/user)
+	if(!crayon || !user)
+		return
+	if(pushed_over)
+		to_chat(user, "<span class='warning'>Right [src] first!</span>")
+		return
+	var/new_appearance = input(user, "Choose a new appearance for [src].", "26th Century Deception") as null|anything in possible_appearances
+	if(!new_appearance || !crayon)
+		return
+	if(!do_after(user, 10, FALSE, src, TRUE))
+		return
+	user.visible_message("<span class='notice'>[user] gives [src] a new look.</span>", "<span class='notice'>Voila! You give [src] a new look.</span>")
+	alpha = 255
+	icon = initial(icon)
+	if(!deceptive)
+		color = "#FFD7A7"
+	switch(new_appearance)
+		if("Assistant")
+			name = "[pick(first_names_male)] [pick(last_names)]"
+			desc = "A cardboat cutout of an assistant."
```

Comment:
Also, as our default title is Civilian, which is different than these other servers that use Assistant, this should probably remain consistent